list/list_resource.go

```diff
@@ -97,6 +98,9 @@ type ListRequest struct {
 	// IncludeResource indicates whether the provider should populate the
 	// [ListResult.Resource] field.
 	IncludeResource bool
+
+	ResourceSchema         fwschema.Schema
+	ResourceIdentitySchema fwschema.Schema
```

Comment:
Seperate note, if we do return these schemas, should we return the actual exported schema types? From `identityschema` and `resource/schema` (not sure we can, cycle import?)

Reply:
To be determined, when we get to the point of handling non-Framework schema types.

On June 17, I'm thinking it's possible that `fwschema.Schema` is the interface to rule them all. Related 💭: https://github.com/hashicorp/terraform-plugin-framework/pull/1165